<!-- serial_bridge.launch.xml -->
 
<launch>

    <!--Serial port settings -->
    <arg name = "port_path" default = "/dev/ttyUSB0"/>
    <arg name = "baud_rate" default = "115200"      />

    <!-- ERP42 parameters -->
    <arg name = "max_speed"           default = "7.00"  />
    <arg name = "max_steering_deg"    default = "28.00" />
    <arg name = "steering_offset_deg" default = "0.00"  />

    <!-- ERP42 serial driver -->
    <node pkg = "erp42_serial" exec = "serial_bridge" name = "erp42_serial_bridge" output = "screen">

        <!--Serial port settings -->
        <param name = "port_path" value = "$(var port_path)"/>
        <param name = "baud_rate" value = "$(var baud_rate)"/>

        <!-- ERP42 parameters -->
        <param name = "max_speed"           value = "$(var max_speed)"          />
        <param name = "max_steering_deg"    value = "$(var max_steering_deg)"   />
        <param name = "steering_offset_deg" value = "$(var steering_offset_deg)"/>

    </node>

</launch>